cmd/swarmctl/cluster/inspect.go

```diff
@@ -52,6 +53,28 @@ func printClusterSummary(cluster *api.Cluster) {
 			fmt.Fprintf(w, "    %s: %s\n", ca.Protocol, ca.URL)
 		}
 	}
+
+	if cluster.Spec.DefaultLogDriver != nil {
+		fmt.Fprintf(w, "DefaultLogDriver\t: %s\n", cluster.Spec.DefaultLogDriver.Name)
```

Comment:
Odd. The rest of these aren't really consistent with anything. These should probably just be raw field names or at least title cased.